### PR TITLE
Requesting a pull to mozilla:v0.5.x from k88hudson:t1632

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/1de2bc540367fef/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/ef3d28513c05b13/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1de2bc540367fef/output.txt

Total script time: 0.82 mins

Server is running on http://pr581.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/ef3d28513c05b13/output.txt

Total script time: 0.26 mins

File: templates/supported/robots/butter/template.js [hunk 1]
```diff
@@ -11,4 +11,4 @@
       }
     }); //Butter
   }, false);
-}( window.Butter, window.EditorHelper ));
+}( window.Butter, window.EditorHelper));
```
Comment (ScottDowne) [inline templates/supported/robots/butter/template.js]: Why remove this space?

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/869d1f481dffbe6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/58c158087cafa04/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/869d1f481dffbe6/output.txt

Total script time: 1.33 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/58c158087cafa04/output.txt

Total script time: 1.33 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/81ca20d2e293dbb/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/14c15d9718b9a66/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/81ca20d2e293dbb/output.txt

Total script time: 0.79 mins

Server is running on http://pr581.butyr.org/templates/test.html

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/14c15d9718b9a66/output.txt

Total script time: 0.73 mins

File: templates/supported/simple/index.html [hunk 2]
```diff
@@ -8,14 +8,18 @@
   <link href="style.css" rel="stylesheet" type="text/css" />
   <script src="../../../src/butter.js" data-butter-exclude></script>
   <script type="text/javascript" src="butter/template.js" data-butter-exclude></script>
+  <script src="../shared/positiondebugger.js"></script>
```
Comment (secretrobotron) [inline templates/supported/simple/index.html]: Might these actually benefit from having `data-butter-exclude` on them, or do you need them for the export too?

**Pull request merged**